linux/runctypes/runc.proto

```diff
@@ -36,3 +36,7 @@ message CheckpointOptions {
 	repeated string empty_namespaces = 6;
 	string cgroups_mode = 7;
 }
+
+message ProcessDetails {
+	string exec_id = 1;
+}
```

Comment:
missing a newline

Reply:
hmm it seems like adding a new line triggers a linting error?

Reply:
:scream_cat: 

I don't see the linting error in travis, just failed tests

Reply:
oh I already updated the commit to address the linting error: 
```
git show --check db849f6e3255c0774be5591cdadc957e4906a91c

linux/runctypes/runc.proto:43: new blank line at EOF.
```

but now looking into the newer failed tests!

Reply:
Oh, my bad, you don't want an empty line at the end of the file, but you want the file to ends with a `\n` :)

Reply:
PTAL @mlaventure